src/main/archetype/ui.frontend.angular/src/app/app.component.ts

```diff
@@ -27,11 +35,27 @@ export class AppComponent {
   items: any;
   itemsOrder: any;
   path: any;
-
+#if ( $enableSSR == "n")
```

Comment:
can `$enableSSR` have only two values? "n" and "y" ? If not (or not sure) then maybe one of the ifs (probably "n") should be used as fallback?

Reply:
yep only these 2 values